libs/designsystem/action-group/src/action-group.component.ts

```diff
@@ -98,9 +101,29 @@ export class ActionGroupComponent implements AfterContentInit {
     action.button.dispatchEvent(event);
   }
 
+  private initializeFromConfig() {
+    if (!this.config) return;
+
+    this._isResizeable = this.config.isResizable;
+
+    if (this.visibleActions === undefined && this.config.defaultVisibleActions !== undefined) {
+      this.visibleActions = this.config.defaultVisibleActions;
+    }
+
+    if (this.config.maxVisibleActions !== undefined) {
+      // Don't overwrite visibleActions value if configured lower than maxVisibleActions:
+      if (!(this.visibleActions < this.config.maxVisibleActions)) {
```

Comment:
Can this be rewritten as 

```suggestion
      if (this.visibleActions >= this.config.maxVisibleActions) {
```

Reply:
Resolving this as the answer is no - this.visibleActions can be undefined, so check needs to look like that